docs/variants/novacustom_ns5x/hardware-matrix.md

```diff
@@ -55,5 +56,6 @@ coreboot port on the Clevo NS51/NS70 laptops.
 |                                | 5. SanDisk Ultra microSD HC I 16 GB + SD Adapter |
 |                                | 6. Logitech, Inc. Keyboard K120                  |
 |                                | 7. 4-port USB Type-C Hub Pro                     |
```

Comment:
In my opinion, there's no sense in putting two different USB-C hubs in one HW matrix. Please remove from the table `4-port USB Type-C Hub Pro`.

Reply:
removed